src/getHumanAge.test.js

```diff
@@ -1,10 +1,58 @@
+/* eslint-disable no-undef */
+/* eslint-disable max-len */
 'use strict';
 
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+  test('should convert cat and dog years to human years correctly for typical cases', () => {
```

Comment:
you already have such test below. Try not to make more than 1 assertion in 1 test.Also, add tests for boundary values